pkg/operator/starter.go

```diff
@@ -418,16 +440,20 @@ func apiServices() []*apiregistrationv1.APIService {
 				VersionPriority:      15,
 			},
 		}
-		ret = append(ret, obj)
+		if groupDisabled[apiServiceGroupVersion.Group] {
+			disabled = append(disabled, obj)
+		} else {
+			enabled = append(enabled, obj)
+		}
 	}
 
-	return ret
+	return enabled, disabled, nil
 }
 
 func apiServicesReferences() []configv1.ObjectReference {
 	ret := []configv1.ObjectReference{}
-	for _, apiService := range apiServices() {
-		ret = append(ret, configv1.ObjectReference{Group: "apiregistration.k8s.io", Resource: "apiservices", Name: apiService.Spec.Version + "." + apiService.Spec.Group})
+	for _, apiService := range apiServiceGroupVersions {
```

Comment:
`apiServicesReferences()` is consumed by `.WithClusterOperatorStatusController` which uses the references to build a mapping of objects. In case a reference has no related object, it gets ignored.

Reply:
ok, it looks like this is purely informational. It doesn't have any impact on the core logic.